parsers/IN_WE.py

```diff
@@ -147,37 +171,28 @@ def format_exchanges_data(
 
 
 def format_consumption_data(
-    data: dict, zone_key: str, target_datetime: datetime
+    data: dict,
+    zone_key: str,
```

Comment:
Not sure why this was included before but zone_key is not used anywhere in this function.

Reply:
Resolved!